src/machine/machine_table.c

```diff
@@ -9358,7 +9358,7 @@ const machine_t machines[] = {
             .min_multi = MACHINE_MULTIPLIER_FIXED,
             .max_multi = MACHINE_MULTIPLIER_FIXED
         },
-        .bus_flags = MACHINE_PCI,
+        .bus_flags = MACHINE_PS2_PCI,
```

Comment:
The TRW page you linked explicitly lists the Dell as _not_ having a PS/2 port though?

Reply:
The Dell is based on one of the Intel boards, and those quite assuredly had PS/2.